milli/src/search/new/ranking_rule_graph/position/mod.rs

```diff
@@ -0,0 +1,133 @@
+use fxhash::{FxHashMap, FxHashSet};
```

Comment:
Query words that match far in a field will be bucketed together if they slightly differ. Query words that match near the front of the field will remain precise enough (see `cost_from_position`).

Reply:
Should we store the aggregated positions on the disk too? From 4 to 16.